pkg/build/builder/sti_test.go

```diff
@@ -72,6 +73,10 @@ func newTestS2IBuilder(config testS2IBuilderConfig) *S2IBuilder {
 func makeBuild() *buildapiv1.Build {
 	t := true
 	return &buildapiv1.Build{
+		ObjectMeta: metav1.ObjectMeta{
```

Comment:
?

Reply:
also +1

Reply:
Builds without these fields are not valid and cause tests to report incorrect values